apps/list.c

```diff
@@ -639,6 +690,8 @@ const OPTIONS list_options[] = {
      "List of message digest algorithms"},
     {"kdf-algorithms", OPT_KDF_ALGORITHMS, '-',
      "List of key derivation and pseudo random function algorithms"},
+    {"rand-algorithms", OPT_RAND_ALGORITHMS, '-',
```

Comment:
They're not really algorithms because I expect two instances of the DRBG algorithm in FIPS and default, for example.  I'm not sure of a better word right now, other than "implementations"

Reply:
Good point, I'll try to come up with an alternative.

Reply:
I've changed the option to `randoms` and the description to _List of random number generators_.